swap_meet/electronics.py

```diff
@@ -1,2 +1,6 @@
-class Electronics:
-    pass
+from swap_meet.item import Item
+class Electronics(Item):
+    def __init__(self, age=None, condition=None, category="Electronics"):
```

Comment:
Same comment here as above in Clothing class, let's directly pass "Electronics" into the super class constructor and remove it from the available initializer parameters.